<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TiTulo_Juego">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>48</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 48pt &quot;Gill Sans MT&quot; rgb(255,255,255);


</string>
    </property>
    <property name="text">
     <string>4 en Linea</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Contador">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="Rein_Cont">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reiniciar Contador</string>
     </property>
    </widget>
    <widget class="QLabel" name="Jug_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jugador Nº 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Con_Vic">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>281</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Contador Victorias:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="Cont_Jug_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLCDNumber" name="Cont_Jug_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="Jug_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jugador Nº 2:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Movimiento">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>541</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Mov_Fichas">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moviemiento de las Fichas:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Mov_Der">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Derecha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Mov_Izq">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Izquierda</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Conf_Pos">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Confirmar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Color">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>971</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 189, 189);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="Colores_Jug_1">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Rojo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Azul</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Verde</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amarillo</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Colores_Jug_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>190</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>Azul</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rojo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Verde</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amarillo</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="Jug_1_Color">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color del Jugador Nº 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Jug_2_Color">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color del Jugador Nº 2:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Conf_Color_Jug_1">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirma Color Jugador Nº 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Conf_Color_Jug_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>190</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirma Color Jugador Nº 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rojo_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Rojo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rojo_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Rojo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Amarillo_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffff00;&quot;&gt;Amarillo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Azul_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Azul&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Verde_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Verde&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Azul_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Azul&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Verde_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Verde&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Amarillo_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffff00;&quot;&gt;Amarillo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Listo_1">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>¡LISTO!</string>
     </property>
    </widget>
    <widget class="QLabel" name="Listo_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>180</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>¡LISTO!</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxConex">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>421</width>
       <height>171</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(189, 189, 189);</string>
     </property>
     <property name="title">
      <string>Conexión</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Puerto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxPuertos">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="ConectarBoton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Conectar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Comienzo_Juego">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>421</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Comenzar Juego</string>
     </property>
    </widget>
    <zorder>Colores_Jug_1</zorder>
    <zorder>Colores_Jug_2</zorder>
    <zorder>Jug_1_Color</zorder>
    <zorder>Jug_2_Color</zorder>
    <zorder>Conf_Color_Jug_1</zorder>
    <zorder>Conf_Color_Jug_2</zorder>
    <zorder>Amarillo_1</zorder>
    <zorder>Amarillo_2</zorder>
    <zorder>Azul_1</zorder>
    <zorder>Azul_2</zorder>
    <zorder>Rojo_1</zorder>
    <zorder>Rojo_2</zorder>
    <zorder>Verde_1</zorder>
    <zorder>Verde_2</zorder>
    <zorder>Listo_1</zorder>
    <zorder>Listo_2</zorder>
    <zorder>groupBoxConex</zorder>
    <zorder>Comienzo_Juego</zorder>
   </widget>
   <widget class="QLabel" name="Fondo_Juego">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1061</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Fondo/Fondos-web.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Fondo_Juego</zorder>
   <zorder>Movimiento</zorder>
   <zorder>Contador</zorder>
   <zorder>Color</zorder>
   <zorder>TiTulo_Juego</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
